src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -4,13 +4,17 @@
     using System.Threading.Tasks;
     using AcceptanceTesting.Support;
 
-    public class DefaultServer : ExternallyManagedContainerServer
+    public class DefaultServer : ServerWithNoDefaultPersistenceDefinitions
     {
-        public override Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor, EndpointCustomizationConfiguration endpointCustomizationConfiguration, Action<EndpointConfiguration> configurationBuilderCustomization)
+        public IConfigureEndpointTestExecution PersistenceConfiguration { get; set; } = TestSuiteConstraints.Current.CreatePersistenceConfiguration();
+
+        public override Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor, EndpointCustomizationConfiguration endpointConfiguration, Func<EndpointConfiguration, Task> configurationBuilderCustomization)
         {
-            return base.GetConfiguration(runDescriptor, endpointCustomizationConfiguration, endpointConfiguration =>
+            return base.GetConfiguration(runDescriptor, endpointConfiguration, async configuration =>
             {
-                configurationBuilderCustomization(endpointConfiguration);
+                await configuration.DefinePersistence(PersistenceConfiguration, runDescriptor, endpointConfiguration).ConfigureAwait(false);
+
+                await configurationBuilderCustomization(configuration);
```

Comment:
```suggestion
                await configurationBuilderCustomization(configuration).ConfigureAwait(false);
```

Reply:
Or consistently remove it if we generally don add it in the ATTs. I don know what the latest state is there